package.json

```diff
@@ -16,8 +18,13 @@
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-register": "^6.26.0",
```

Comment:
Was this dependency used anywhere? If no, it will need to be removed.

Reply:
Babel-preset was used but babe-register was not. Removing babel-register asap